resolwe/flow/tests/test_executors.py

```diff
@@ -27,7 +27,8 @@ class DockerExecutorTestCase(unittest.TestCase):
     @mock.patch('os.chmod')
     @mock.patch('os.chdir')
     @mock.patch('resolwe.flow.executors.Data.objects.filter')
-    def test_run_in_docker(self, data_filter_mock, chdir_mock, chmod_mock, mkdir_mock):
+    @mock.patch('resolwe.flow.executors.Data.objects.get')
```

Comment:
Was this mock object accidentally added since it is not used inside the test function?